src/DiffEqOperators.jl

```diff
@@ -53,8 +56,9 @@ export AnalyticalJacVecOperator, JacVecOperator, getops
 export AbstractDerivativeOperator, DerivativeOperator,
        CenteredDifference, UpwindDifference
 export DirichletBC, Dirichlet0BC, NeumannBC, Neumann0BC, RobinBC, GeneralBC, MultiDimBC, PeriodicBC,
-       MultiDimDirectionalBC, ComposedMultiDimBC,
-       compose, decompose, perpsize
+       MultiDimDirectionalBC, ComposedMultiDimBC
+export compose, decompose, perpsize, c2l
```

Comment:
we shouldn't export a short name like `c21`. I'll see what that's all about further down I assume

Reply:
no longer exported; renamed to `cartesian_to_linear`